packages/app/components/trending/trending-creators-list.md.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState, useCallback } from "react";
+import { ListRenderItemInfo } from "react-native";
+
```

Comment:
no problem with this file but I think we won't be using trending creators for the time being.

Reply:
yeah I know, I have been tangled for a while to change this file 🤣, but I thought I might use it in the future, so I improved. (maybe it's my OTD I want consistency.)